rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -132,7 +132,7 @@ ImageInfo::ImageInfo(
         _height(height_),
         _color_planes(planes),
         _batch_size(batches),
-        _data_size(width_ * height_ * _batch_size * planes),
+        _data_size((uint64_t)width_ * height_ * _batch_size * planes),
```

Comment:
It will be better to do a static cast here
`(static_cast<uint64_t>(width_ * height_ * _batch_size * planes))`